_guides/integrations-3.md

```diff
@@ -1,30 +1,35 @@
-description: Bypass manual transfer of .csv files by connecting your database directly to OpenLattice. This guide is recommended for individuals who are interested in setting up integrations into the OpenLattice platform where data is transferred directly and securely from database to database. Reach out to support@openlattice.com if you need any assistance with your integration.
+title: Integration Pipeline: Secure Data Transfers & Speed
+description: Maximize both security and performance by using our **Launchpad** application to transfer data directly and securely from your database directly into OpenLattice. This is the recommended data pipeline for using the OpenLattice platform. Reach out to support@openlattice.com if you need any assistance with your integration.
 weight: 1
 ---
 
 * TOC
 {:toc}
 
-## 1. Overview 
-The previous tutorial, [Integrations Part 2 – Integration Scripts & Reading Data from .csv files](/guides/integrations-2/), walked one through how to run integrations from one's own computer from individual .csv files, one by one, into the OpenLattice platform. However, it may often be more desirable to transfer data, via either whole tables or subsets of tables, directly from your computer or SQL server and onto an OpenLattice server for later integration. 
+## 1. Balancing security & performance 
+The desire to maximize security and maintain control over one's data should not prevent one from taking full advantage of OpenLattice's ability to perform fast and accurate data integrations for you.
+* Writing and running integrations on your own server (integration tutorials 1 and 2) eliminates the need for any data transfers, but may impact your computer system's performance and memory depending on your processing power, especially for large files or real-time, recurring integrations.
+* Running integrations on OpenLattice servers can be faster by orders of magnitude.
 
-Your data would be stored in a private database accessible only to you - **please contact <a href="mailto:support@topenlattice.com">support@openlattice.com</a> for arrangements.** Later, one can still write your own integration script if desired and also put it onto OpenLattice servers to run automatically. A tutorial for this final step is being developed. 
+## 2. Launchpad
+Data transfers are obviously integral to using the OpenLattice platform, and we take very seriously the need to provide the highest level of security and transparency. For these reasons we have developed a secure and completely open access solution.
 
-Some advantages of this workflow include:
+Our [Launchpad](https://github.com/openlattice/launchpad) application eliminates the need for, and questionable security of, email file transfers by allowing you to transfer whole tables or subsets of tables, from your computer or SQL server directly to an OpenLattice server for later integration. Some advantages of this workflow include:
```

Comment:
Maybe "eliminates the need for, and insecurity of, ..." since we're not eliminating security